### PR TITLE
Import Export Module: Tweak - The Import Export tools tab is not temporary - ED-2675,ED-2500

### PR DESCRIPTION
**Pull request by rotemee**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

File: core/app/modules/import-export/module.php [hunk 1]
```diff
@@ -122,82 +122,31 @@ private function render_temp_import_export_content() {
...
 				'link' => [
-					'url' => '#',
+					'url' => 'https://go.elementor.com/wp-dash-import-export-import-flow',
 					'text' => __( 'Learn More', 'elementor' ),
 				],
 			],
 		];
 
-		$info_text_link = sprintf( '<a href="#">%s</a>', __( 'Click here.', 'elementor' ) );
+		$front_page_id = get_option( 'page_on_front' );
```
Comment (kobizz) [inline core/app/modules/import-export/module.php:131]: What should happen if there is no page on front?

File: core/app/modules/import-export/module.php [hunk 2]
```diff
@@ -122,82 +122,31 @@ private function render_temp_import_export_content() {
...
+
+		$front_page_link_Action = sprintf( '/post.php?post=%s&action=elementor#e:route:panel/history/revisions', $front_page_id );
```
Comment (kobizz) [inline core/app/modules/import-export/module.php:133]:
```suggestion
		$front_page_link_action = sprintf( '/post.php?post=%s&action=elementor#e:route:panel/history/revisions', $front_page_id );
```

**Review by kobizz — CHANGES_REQUESTED** [review]

File: core/app/modules/import-export/module.php (hunk 1 above)
Comment (rotemee) [reply core/app/modules/import-export/module.php]: As agreed with @DorShahar, the info link was removed.

File: core/app/modules/import-export/module.php (hunk 2 above)
Comment (rotemee) [reply core/app/modules/import-export/module.php]: The info link was totally removed.

**Pull request merged**